README.md

```diff
@@ -45,13 +45,19 @@ messenger:
                                        # - "markdown" (for Mattermost)
                                        # - "slack" (for Slack)
   sender:
-    username: "@ubmrellio/gbot"        # Sender's display name
+    username: "@umbrellio/gbot"        # Sender's display name
     icon: "<icon url>"                 # Sender's icon url
 gitlab:
   token: "<TOKEN>"                     # GitLab Private Access Token
   url: "<gitlab api url>"              # Gitlab API base url
   projects:                            # List of your project ids
   - 42
+  groups:                              # List of your project’s groups
```

Comment:
Shall we declare the config this way probably? 🤔 
```yaml
groups:
- id: 4
  excluded: [1, 2, 3]
```

Reply:
'cause excluded projects relate to a group, but the group declares near in another key, which might confuse